development/build/set-environment-variables.js

```diff
@@ -189,28 +189,24 @@ function getPhishingWarningPageUrl({ variables, testing }) {
         }/`;
   }
 
-  // We add a hash/fragment to the URL dynamically, so we need to ensure it
-  // has a valid pathname to append a hash to.
-  const normalizedUrl = phishingWarningPageUrl.endsWith('/')
-    ? phishingWarningPageUrl
-    : `${phishingWarningPageUrl}/`;
```

Comment:
This was error prone if the phishing page ends up ever being something like `https://domain.com/page-with-trailing-slash.html`. The logic is replaced with `phishingWarningPageUrlObject.toString();` below, which adds a trailing slash to the end of the domain if it is missing and there URL has nothing after it.

Reply:
:+1: